tests/llm/test_openai/test_stream.py

```diff
@@ -29,25 +30,21 @@ def test_iterable_model(model, mode, stream, client):
         assert isinstance(m, UserExtract)
 
 
-@pytest.mark.parametrize("model, mode, stream", product(models, modes, [True, False]))
+@pytest.mark.parametrize("model, mode", product(models, modes))
 @pytest.mark.asyncio
-async def test_iterable_model_async(model, mode, stream, aclient):
-    aclient = instructor.patch(aclient, mode=mode)
+async def test_iterable_model_async(model, mode, aclient):
+    aclient = instructor.from_openai(aclient, mode=mode)
```

Comment:
Removed stream=True and false tests since I migrated the underlying call to use create_iterable so that we have a more consistent handling of iterables.